index.html

```diff
@@ -10,9 +10,47 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
+    <!-- preload product images before the bundle loads -->
```

Comment:
A bit ugly, but it's the most surefire way of ensuring that we preloaded our product images before the JavaScript bundle even loads.